tpp-app/src/home/pages/CycleHistoryScreen.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from 'react';
+import {SafeAreaView, Text, StyleSheet, View, TouchableOpacity, ImageBackground} from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import background from '../../../ios/tppapp/Images.xcassets/SplashScreenBackground.imageset/watercolor-background.png';
+import {ExpandedHistoryCard} from '../components/CycleHistory';
+import CycleService from '../../services/cycle/CycleService';
+import {GETStoredYears} from '../../services/utils/helpers';
+
+function Header({navigation}){
+    return(
+    <View style={ styles.headerContainer}>
+        <View
+        style={[styles.nonCenterComponent]}
+        >
+            <TouchableOpacity 
+                onPress={() =>navigation.goBack()}
+            >
+                <Icon name="keyboard-arrow-left" size={36} color={"#5A9F93"}/>
+            </TouchableOpacity>
+        </View>
+        <Text style={styles.headerText}> Period History </Text>
+        <View style={styles.nonCenterComponent}><Text></Text></View>
+    </View>
+    );
+}
+
+function YearButton({year, selectedYear, setSelectedYear}){
+
+    let backgroundColor = year == selectedYear ? "#B31F20" : "#FFFFFF";
+    let textColor = year == selectedYear ? "#FFFFFF" : "#C4C4C4";
+    let border = year == selectedYear ? null : styles.buttonBorder;
```

Comment:
```suggestion
    let backgroundColor = (year === selectedYear) ? "#B31F20" : "#FFFFFF";
    let textColor = (year === selectedYear) ? "#FFFFFF" : "#C4C4C4";
    let border = (year === selectedYear) ? null : styles.buttonBorder;
```
Best practice to use triple equality comparators